integration-tests/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/it/helpers/TestManifest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.it.helpers;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.ResourceLoader;
+import org.yaml.snakeyaml.Yaml;
+
+public class TestManifest {
+
+  private final List<Map<String, Object>> contents;
+  private ObjectMapper objectMapper = new ObjectMapper();
+
+  public TestManifest(String path) {
```

Comment:
Is there a reason we need to return a class `TestManifest` here instead of loading the resources and returning a `List<Map<String, Object>>` directly?

Obviously for production code (that has lots of tests on it) it might make sense to encapsulate this, but here it feels like the indirection again makes it a bit harder to see what's going on when trying to debug the tests.